tests/sentry/integrations/vercel/test_integration.py

```diff
@@ -89,6 +89,13 @@ def assert_setup_flow(self, is_team=False):
             "installation_id": "my_config_id",
             "installation_type": installation_type,
             "webhook_id": "webhook-id",
+            "configurations": {
```

Comment:
@MeredithAnya I think we should have a test where we install the integration for two different orgs

Reply:
yeah i need to add that